pmm-app/package.json

```diff
@@ -11,7 +11,7 @@
   "main": "module.js",
   "scripts": {
     "build": "grafana-toolkit plugin:build",
-    "test": "grafana-toolkit plugin:test --updateSnapshot",
+    "test": "grafana-toolkit plugin:test",
```

Comment:
Good fix, but we'll also need command to update snapshots, so please add additional script for it

Reply:
I don't get it: the way it was before, `npm test` would pass even if the snapshots are different. I'm sure that's not the way snapshots have been designed.

Reply:
Anyway, added another script with --updateSnapshot